doc/connectivity/resting_state_networks.rst

```diff
@@ -27,15 +27,16 @@ Multi-subject ICA: CanICA
 Data preparation: retrieving example data
 -----------------------------------------
 
-We will use sample data from the `ADHD 200 resting-state dataset
-<http://fcon_1000.projects.nitrc.org/indi/adhd200/>`_ has been
-preprocessed using `CPAC <http://fcp-indi.github.io/>`_. We use nilearn
+We will use down-sample data from the `children resting-state dataset
+<https://openneuro.org/datasets/ds000228/versions/1.0.0>`_ has been
+preprocessed using `FMRIPrep <https://github.com/poldracklab/fmriprep>`_.
+We use nilearn
```

Comment:
Again, I think we might want to link OSF here. It gives the exact fMRIPrep "methods section" as well as the Nilearn commands used in downsampling.